controllers/app.controllers.js

```diff
@@ -0,0 +1,6 @@
+const endpointsJson = require("../endpoints.json");
+
+exports.getApi = (req, res) => {
+  console.log(endpointsJson, "<= endpointsJson");
```

Comment:
console.logs that are no longer needed should be removed before code is pushed up to github